### DISCUSSION TITLE
"Convert to issue" button in issue task list has no confirmation

### DISCUSSION THREAD

**Discussion by victorlin** [community/community#14011] [Projects and Issues]
For an issue with a task list, the individual tasks each have a clickable **Convert to issue** icon upon hover:

<img width="400" alt="Convert to issue icon" src="https://user-images.githubusercontent.com/13424970/161144171-d16c2f71-3cd4-495b-b428-4cd33f807f9a.png">

When clicking on this, a new issue in the same repository is automatically created using the task text as the title and no description. I think this behavior is too direct and surprises many users, including myself:

1. This makes it easy to accidentally create an issue.
2. Most users would want to update the title and/or add a description after it's created like this, which is an extra operation.

**Comment by victorlin** [comment]
I would like to see some sort of confirmation and customization popup, similar to the **Reference in new issue** feature on a comment:

<img width="400" alt="Reference in new issue popup" src="https://user-images.githubusercontent.com/13424970/161144517-44cf8463-84bf-49f9-af8e-28867b2402c3.png">

When holding <kbd>shift</kbd>, the **Convert to issue** turns into **Open convert to issue** which is better but not great because it opens a new window (at least in Microsoft Edge browser, where that is the universal behavior <kbd>shift</kbd>+click).

**Reply by BenjSD** [reply]
@victorlin 
you can also use : 
- `Ctrl` + `Shift` + `click `= open in new tab
- `Alt` + `click `= open in current tab

(but I'm searching the documentation about this ... maybe there are other tips ?)

**Reply by BenjSD** [reply]
I've just found it : https://docs.github.com/en/get-started/using-github/keyboard-shortcuts#issues-and-pull-requests

**Comment by MakePixelsWork** [comment]
I disagree. Having to fill out all fields is too mandatory. I prefer the current option. As a projectmanager, I can convert tasks into issues right away. You title has to be descriptive anyway. If I want to edit its contents, I will open the actual issue myself and add to it. If each issue has a screen that opens each time you convert to an issue, that's just more annoying.

**Reply by SalcedoAllan** [reply]
Where did this option go? Tried converting task list items to issues and the option to do so is no longer there.

**Reply by nexushoratio** [reply]
@SalcedoAllan Is this still a problem for you?  I've just been reading through a bunch of these posts with the word *task* in them, and it seems like it was broken for a bit, but was working again.

**Reply by nexushoratio** [reply]
I just mentioned over in #6898 that I would like to metadata from the parent issue transferred to the child issue.  However, I think that would require a confirmation pop-up.

If the fields had reasonable defaults that did not need to be edited most of the time (e.g., project level defaults), would you be all right with the UX changing from *click-on-circle* to *click-on-circle-THEN-click-on-OK-button*?
